src/org/sosy_lab/cpachecker/core/CPAchecker.java

```diff
@@ -402,14 +450,11 @@ public CPAcheckerResult run(
         }
       }
 
```

Comment:
Where is printConfigurationWarnings() from origin?

Reply:
I've moved it into constructor.